opencti-platform/opencti-front/src/utils/tests/test-render.tsx

```diff
@@ -16,8 +16,25 @@ interface CreateUserContextOptions {
   entitySettings?: unknown,
   platformModuleHelpers?: unknown,
   schema?: unknown,
+  overviewLayoutCustomization?: unknown,
 }
 
+const fakeOverviewLayoutCustomization = new Map();
+const threatActorIndividualConfiguration = new Map([
+  ['details', 6],
```

Comment:
why did you remove 'order' and 'width' ?
```suggestion
  ['details', { order: 1, width : 6 }],
```

Reply:
Because the model has evolved.
The order and width would be stored in the database but the configuration used for the template will be based on entries sorted by order with the value of the width.

Reply:
ok thanks